packages/vuetify/src/mixins/themeable/index.ts

```diff
@@ -65,10 +63,10 @@ const Themeable = Vue.extend<Themeable>().extend({
       } else if (this.light === true) {
         // explicitly light
         return false
-      } else {
-        // inherit from parent, or default false if there is none
-        return this.theme.isDark
       }
+
+      // inherit from parent, or default false if there is none
```

Comment:
Comment is wrong now

Reply:
this has to be updated anyways. I don't want components to have the possibility to fail because of $vuetify not being defined.